test/integration/unregistered_user_can_view_categories_of_items_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.